.changeset/great-parrots-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/web-styles': patch
+---
+
+Moves /.stylelintrc.js to /web-styles/.stylelintrc, so it works correctly with the IDE
```

Comment:
A change like this is irrelevant to consumers of our packages, therefore I'd not add it to the changelog. Can you delete this changeset?